commit.go

```diff
@@ -40,6 +40,68 @@ func (c *Commit) RawMessage() string {
 	return ret
 }
 
+func (c *Commit) RawHeader() string {
+	ret := C.GoString(C.git_commit_raw_header(c.cast_ptr))
+	runtime.KeepAlive(c)
+	return ret
+}
+
+// ContentToSign returns the content that will be passed to a signing function for this commit
+func (c *Commit) ContentToSign() string {
+	return c.RawHeader() + "\n" + c.RawMessage()
+}
+
+// CommitSigningCb defines a function type that takes some data to sign and returns (signature, signature_field, error)
+type CommitSigningCb func(string) (string, string, error)
```

Comment:
```suggestion
// CommitSigningCb defines a function type that takes some data to sign and returns (signature, signature_field, error)
type CommitSigningCb func(string) (signature, signatureField string, err error)
```

welp, i missed this. it should make the godoc a bit better.

Reply:
Fixed this and also changed the name of the type/field to be more appropriate (based on feedback from other PR)